pkg/kubelet/server/server_test.go

```diff
@@ -566,7 +566,7 @@ func TestAuthFilters(t *testing.T) {
 
 	attributesGetter := NewNodeAuthorizerAttributesGetter(authzTestNodeName)
 
-	for _, tc := range AuthzTestCases() {
+	for _, tc := range AuthzTestCases(false) {
```

Comment:
E2E tests can probably wait for beta, but it would be good to at least have unit test coverage of the authz fallback logic:

- first subresource is authorized, second is not
- first subresource is not authorized, second is
- neither subresources are authorized

Reply:
Added a new unit test.